File: C Programming/Assignments/Assignment 6/func with no return type and with para/Assignment 1/Q2/Q2.c
//Write a program to check given 3 digit number is pallindrome or not.

#include<stdio.h>

void pallindrome(int);
void main(){
	
	//variable declaration and initialization
	int digit = 313;
	
	pallindrome(digit);
	
}

void pallindrome(int digit){

	int first_digit, last_digit;
	
	//extracting first and last digit from given digit
	first_digit = digit/100;
	last_digit = digit%10;
	
	//checking extracted digits were equal or not
	if(first_digit == last_digit)
		printf("%d is an pallindrome number",digit);
	else
		printf("%d is not an pallindrome number",digit);	
	
}

File: FirstBit Workspace/C Programming/Assignments/java_to_c_connectivity/Assignment 1/Q2/Q2.java
//Write a program to check given 3 digit number is pallindrome or not.

public class Q2 {
    public static void main(String[] args) {
        // variable declaration and initialization
        int digit = 313;
        int firstDigit, lastDigit;

        // extracting first and last digit from given digit
        firstDigit = digit / 100;
        lastDigit = digit % 10;

        // checking extracted digits were equal or not
        if (firstDigit == lastDigit)
            System.out.println(digit + " is a palindrome number");
        else
            System.out.println(digit + " is not a palindrome number");
    }
}
